android-components/components/service/firefox-accounts/src/main/java/mozilla/components/service/fxa/sync/WorkManagerSyncManager.kt

```diff
@@ -562,7 +562,13 @@ internal fun getSyncState(context: Context): String? {
         .getString(SYNC_STATE_KEY, null)
 }
 
-internal fun setLastSynced(context: Context, ts: Long) {
+/**
+ * Saves the lastSyncedTime to the shared preferences
+ *
+ * @param context the context
+ * @param ts - the last synced time in milliseconds
+ */
+fun setLastSynced(context: Context, ts: Long) {
```

Comment:
Thanks for adding some context to this func. Perhaps if `ts` was renamed to something like lastSyncTimeMs we don't need the doc?

Reply:
it seems a good idea to rename that parameter to a more suggestive one, but it still needs the documentation as the function became public